src/App/Fossa/Container/AnalyzeNative.hs

```diff
@@ -115,11 +117,23 @@ analyze ::
   m ()
 analyze cfg = do
   logInfo "Running container scanning with fossa experimental-scanner!"
-  parsedSource <- runDockerEngineApi $ parseContainerImageSource (unImageText $ imageLocator cfg)
+
+  let host :: Text
+      host = (dockerHost cfg)
+
+  let defaultArch :: Text
+      defaultArch = arch cfg
```

Comment:
Why are we pre-declaring the type of these variables? These can be inferred, my preference would be to just let them be inferred.